config/storage.yml

```diff
@@ -7,28 +7,16 @@ local:
   root: <%= Rails.root.join("storage") %>
 
 # Use rails credentials:edit to set the AWS secrets (as aws:access_key_id|secret_access_key)
-# amazon:
-#   service: S3
-#   access_key_id: <%= Rails.application.credentials.dig(:aws, :access_key_id) %>
-#   secret_access_key: <%= Rails.application.credentials.dig(:aws, :secret_access_key) %>
-#   region: us-east-1
-#   bucket: your_own_bucket
+amazon_dev:
+  service: S3
+  access_key_id: <%= Rails.application.credentials.aws[:access_key_id] %>
+  secret_access_key: <%= Rails.application.credentials.aws[:secret_access_key] %>
+  region: <%= Rails.application.credentials.aws[:region] %>
+  bucket: <%= Rails.application.credentials.aws[:dev][:bucket] %>
 
-# Remember not to checkin your GCS keyfile to a repository
-# google:
-#   service: GCS
-#   project: your_project
-#   credentials: <%= Rails.root.join("path/to/gcs.keyfile") %>
-#   bucket: your_own_bucket
-
-# Use rails credentials:edit to set the Azure Storage secret (as azure_storage:storage_access_key)
-# microsoft:
-#   service: AzureStorage
-#   storage_account_name: your_account_name
-#   storage_access_key: <%= Rails.application.credentials.dig(:azure_storage, :storage_access_key) %>
-#   container: your_container_name
-
-# mirror:
-#   service: Mirror
-#   primary: local
-#   mirrors: [ amazon, google, microsoft ]
+amazon_prod:
+  service: S3
+  access_key_id: <%= Rails.application.credentials.aws[:access_key_id] %>
```

Comment:
In general I prefer to put all this data in an ENV var, because is easier to deal with it later in heroku. So you'd probably need to follow the next steps:

1- In the Gemfile add this gem to **group :development, :test**: `gem 'dotenv-rails', '~> 2.7.2'`
2- Add a new line in the **.gitignore** with `.env` (in order to avoid to commit that local file)
3- Change this file to have:
```
access_key_id: <%= ENV['ACCESS_KEY_ID'] %>
secret_access_key: <%= ENV['SECRET_ACCESS_KEY_ID'] %>
region: <%= ENV['REGION'] %>
bucket: <%= ENV['BUCKET] %>
```
4- Add those ENV vars values in Heroku, inside **production** app settings

Reply:
Wdyt @federicoaldunate ?

Reply:
I think it's a good idea. We didn't deploy to Heroku yet but I like the idea of doing it now so it will be easier then